pkg/planner/core/find_best_task.go

```diff
@@ -2688,6 +2677,19 @@ func convertToTableScan(ds *logicalop.DataSource, prop *property.PhysicalPropert
 		if candidate.path.Index != nil && candidate.path.Index.VectorInfo != nil && !candidate.isMatchProp {
 			return base.InvalidTask, nil
 		}
+	} else {
+		if isTiFlashPath && config.GetGlobalConfig().DisaggregatedTiFlash || isTiFlashPath && ds.SCtx().GetSessionVars().IsTiFlashCopBanned() {
+			// prop.TaskTp is cop related, just return base.InvalidTask.
+			return base.InvalidTask, nil
+		}
+		if isTiFlashPath && candidate.isMatchProp && ds.TableInfo.GetPartitionInfo() != nil {
+			// TableScan on partition table on TiFlash can't keep order.
+			return base.InvalidTask, nil
+		}
```

Comment:
Does here mean that other columnar indexes can be used when the vector index is already applied?

Reply:
Inverted index is compatible with other columnar indexes.